nix/proto-lens-pkg.nix

```diff
@@ -0,0 +1,23 @@
+{ mkDerivation, base, bytestring, containers, deepseq, ghc-prim
```

Comment:
Let's remove everything related to nix from this repo. It will improve chances of merging pull to mainstream. Plus it not uses haskell.nix. Let's to to return to this overlay hell again.

Reply:
I'll remove it from this branch before opening PR to google. I want to keep it in our master because I need to work with code somehow.